### ISSUE TITLE
Add labels to inputs for accessibility

### ISSUE THREAD

**Issue by HaSistrunk** [sarapis/orservices#68]
## Is your feature request related to a problem? Please describe.
Input fields should have a description that is explicitly associated with the field to make sure that users of assistive technologies will know what the field is for. Placeholder text (eg. "Search for service") is not a replacement for labels, since it disappears once the input is selected and has low color contrast (See [this article on placeholders and accessibility](https://www.deque.com/blog/accessible-forms-the-problem-with-placeholders/)).  

I'm only seeing this issue in the "Search for Service", "Search for Location", and "Search for Organization" inputs. Looks good in the Suggest a Change form and `/contacts` and `/facilities` pages with search.

## Describe the solution you'd like
I'd suggest one of two solutions:
1. Replace placeholder text with visible `<label>` elements above the input
2. Add `<label>` elements that are hidden from sighted users, but will be read by assistive tech.

Link labels to their inputs in the markup. See markup examples for implementation on the [A11y Style Guide](https://a11y-style-guide.com/style-guide/section-forms.html#kssref-forms-search). 

For CSS to create a `.visually-hidden` class that is accessible to screen readers, see this article from The A11y Project on [how to hide content](https://www.a11yproject.com/posts/2013-01-11-how-to-hide-content/).

## Additional context
This accessibility guideline is specified in [WCAG 2.x Level A (SC 3.3.2)](https://www.w3.org/WAI/WCAG21/quickref/#labels-or-instructions)
